src/main.rs

```diff
@@ -20,6 +20,11 @@ use interpreter::Interpreter;
 fn main() {
     let args: Vec<String> = env::args().collect();
 
+    if args[1].eq("--version") || args[1].eq("-v") {
+        println!("Europa Lang v0.0.0"); // todo: rust compiler version
```

Comment:
You can use Cargo env's to do this.
```suggestion
        println!("Europa Lang v{}", env!("CARGO_PKG_VERSION"));
```

Reply:
alright, thank you!